src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx

```diff
@@ -93,6 +93,10 @@ To install the Microsoft SQL Server integration, follow the instructions for you
 3. Go to `C:\Program Files\New Relic\newrelic-infra\integrations.d\` and rename the `mssql-config.yml.sample` file to `mssql-config.yml`.
 4. Edit the `mssql-config.yml` configuration file with your favorite editor. Check out some [configuration file examples.](#examples).
 
+<Callout variant="important">
+  If using a Group Managed Service Account, gMSA, for SQL Server access, the agent needs to run as the gMSA.
```

Comment:
```suggestion
  The agent needs to run as the gMSA if it uses a group managed service account for SQL Server access.
```

Reply:
@phillipsj can you explain what this call out means, actually? I'm understanding that the agent itself needs to be configured to access a group managed service account (therefore letting it run as a gMSA) if the SQL server itself is managed by a group-type account on the mySQL-side? Is that correct?

Reply:
@akristen I can see that being confusing. If you require services to access a SQL Server using a gMSA, then the service, like the New Relic Agent, needs to be running as that account.